insightfinder/InsightFinderDashboard--2022-08-01T17_11_21.json

```diff
@@ -0,0 +1 @@
r detected incidents from InsightFinder.\n\nAbout InsightFinder - InsightFinder Unified Intelligence Engine (UIE) platform provides human centered AI solutions for identifying incident root causes, predicting and preventing production incidents. Visit us at www.insightfinder.com to learn more.\n\nhttps://insightfinder.com/datadog-integration/","background_color":"transparent","font_size":"14","text_align":"left","vertical_align":"top","show_tick":false,"tick_pos":"50%","tick_edge":"left","has_padding":false},"layout":{"x":0,"y":0,"width":12,"height":1}}]},"layout":{"x":0,"y":0,"width":12,"height":2}},{"id":6865066213940010,"definition":{"title":"Predicted Incidents Timeline","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"search":{"query":""},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[{"facet":"severity","sort":{"aggregation":"count","order":"desc"},"limit":10}]}],"response_format":"timeseries","style":{"palette":"orange","line_type":"solid","line_width":"normal"},"display_type":"line"}]},"layout":{"x":0,"y":0,"width":12,"height":2}},{"id":159134579140046,"definition":{"title":"Total Predicted Incidents","title_size":"16","title_align":"left","time":{},"type":"query_value","requests":[{"formulas":[{"formula":"query1"}],"response_format":"scalar","queries":[{"search":{"query":""},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[]}]}],"autoscale":true,"precision":2,"timeseries_background":{"type":"bars","yaxis":{}}},"layout":{"x":0,"y":2,"width":2,"height":2}},{"id":396654075496888,"definition":{"title":"Top 10 Incidents","title_size":"16","title_align":"left","type":"toplist","requests":[{"formulas":[{"formula":"query1","limit":{"count":10,"order":"desc"}}],"response_format":"scalar","queries":[{"search":{"query":""},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[{"facet":"severity","sort":{"aggregation":"count","order":"desc"},"limit":10}]}]}]},"layout":{"x":2,"y":2,"width":5,"height":2}},{"id":2279940465993678,"definition":{"title":"","title_size":"16","title_align":"left","type":"query_table","requests":[{"formulas":[{"formula":"query1","conditional_formats":[],"limit":{"count":10,"order":"desc"},"cell_display_mode":"bar"}],"response_format":"scalar","queries":[{"search":{"query":""},"data_source":"incident_analytics","compute":{"aggregation":"count"},"name":"query1","indexes":["*"],"group_by":[{"facet":"severity","sort":{"aggregation":"count","order":"desc"},"limit":10}]}]}],"has_search_bar":"never"},"layout":{"x":7,"y":2,"width":5,"height":2}}],"template_variables":[],"layout_type":"ordered","is_read_only":false,"notify_list":[],"reflow_type":"fixed","id":"mp4-6pb-gku"}
```

Comment:
Still waiting for the maintainer here, but a few comments:

- You could format this file so it will be easier to read
- Dashboards should be added in the assets/dashboards folder
- Dashboards should be added to the manifest file https://docs.datadoghq.com/developers/integrations/new_check_howto/?tab=manifest